R/remove_constants.R

```diff
@@ -52,7 +52,7 @@ remove_constants <- function(data, cutoff = 1L) {
   dat     <- data %>% janitor::remove_constant()
   removed <- setdiff(colnames(data), names(dat))
   if (length(removed) > 0L) {
-    add_this <- paste(removed, sep = ", ")
+    add_this <- toString(removed)
```

Comment:
`paste(sep =)` on a vector has no effect. Can you clarify what was intended here please? The proposed change causes a test failure so it's unclear to me what the expected behaviour was.

Reply:
`paste` was a replacement for `glue::glue_collapse()` which uses `sep`. I mistakenly forgot to replace `sep`, with `collaspe` as I intend to convert the `removed` vector into a comma-separated list of column names. This is suitable for the display in `print_report()`.

Previously, I had the below in the corresponding test file:

```{r}
expect_identical(report[["constant_columns"]], c("event_name", "country_code", "country_name"))
```

I have corrected this to account for the comma-separated list of column names returned by `toString()`:

```{r}
expect_identical(report[["constant_columns"]], "event_name, country_code, country_name")
```